content/tools/herman.md

```diff
@@ -97,3 +98,23 @@ and keep everything in one place.
   collections.all | getData('data.accoutrement_modules'),
   title='Integrate with Accoutrement'
 ) }}
+
+{{ layout.title('OSS Sponsors') }}
+[comment]: <> (## Great Horned Owl)
+
+[comment]: <> (## Blue-Footed Booby)
```

Comment:
@SondraE because of the widget showing all levels together, do we need to keep these headings commented out?